spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -856,7 +856,7 @@ def add_all_to_nwbfile(
             metadata: dict = None,
             write_as_lfp: bool = False,
             write_scaled: bool = False
-    ):
+        ):
```

Comment:
However, though its not a big deal (just something I would change back the next time I do something in this repo) - the indentation level you change here is an E125 violation in PEP8.

Reply:
yes, there's something weird in my VScode that makes me do this every time, I'll fix it